styles/style.scss

```diff
@@ -231,14 +236,15 @@ img {
 
 iframe.embeddedform {
     display: none;
-    // override the defailt width and height HTML attributes
+    // override the default width and height HTML attributes
     width: 100%;
     height: 100%;
 }
 
 @media (min-width: 768px) {
     .details-container{
-        flex-direction: row;
+        display: flex;
```

Comment:
I'm not sure this is necessary. the `details-container` class already has the `display: flex`, and `flex-wrap` properties. this is just a media query to change it from `flex-direction: column;` to `flex-direction: row;` on devices bigger than `768px`